include/gatekeeper_net.h

```diff
@@ -21,6 +21,7 @@
 
 #include <stdint.h>
 
+#define GATEKEEPER_RSS_KEY_LEN 40
```

Comment:
Where does this constant come from? A comment is likely warranted here. Shouldn't the number `40` be between parentheses like other constants?